networklibrary/src/main/java/com/eis/communication/network/Commands/AddResource.java

```diff
@@ -0,0 +1,27 @@
+package com.eis.communication.network.Commands;
+
+import com.eis.communication.network.SMSNetworkManager;
+
+/**
+ * Command to add a resource
+ *
+ * @author Edoardo Raimondi
+ */
+public class AddResource implements Command {
+
+    /*Using strings to be coherent to SMSNetDictionary class*/
+    private String key;
+    private String value;
+    private SMSNetworkManager networkManager;
+
+    public AddResource(String key, String value, SMSNetworkManager networkManager){
```

Comment:
Why are these commands specific to sms? Couldn't you ask for a `NetworkDictionary` and a `NetworkBroadcaster`?

Reply:
I actually proposed this exact change twice, and I still want this exact change to happen, so yes, I'll do it.